lib/facter/simplib_sysctl.rb

```diff
@@ -0,0 +1,46 @@
+# _Description_
```

Comment:
@trevor-vaughan you said you had a better way to do this. It should go into another PR and ticket.